src/legacy/ui/public/field_editor/components/field_format_editor/editors/color/color.test.js

```diff
@@ -21,7 +21,7 @@ import React from 'react';
 import { shallowWithIntl } from 'test_utils/enzyme_helpers';
 
 import { ColorFormatEditor, ColorFormatEditorComponent } from './color';
-import { DEFAULT_COLOR } from '../../../../../../../core_plugins/kibana/common/field_formats/types/color_default';
+import { DEFAULT_COLOR } from '../../../../../../../../plugins/data/public';
```

Comment:
I think that now that we're importing DEFAULT_COLOR straight from `data`, it needs a better name. What is default color of?

Reply:
Good point. Agree. The constant is changed